File: test/data/usecase/src/py/PhylSowingDateCorrection.py
if (Latitude < 0):
    if (SowingDay > SDsa_sh):
        FixPhyll = P * (1 - Rp * min(SowingDay - SDsa_sh, SDws))
    else: FixPhyll = P
else:
    if (SowingDay < SDsa_nh):
        FixPhyll = P * (1 - Rp * min(SowingDay, SDws))
    else: FixPhyll = P

File: test/data/usecase/src/cs/PhylSowingDateCorrection.cs
double FixPhyll;
if (Latitude < 0)
{
	if (SowingDay > SDsa_sh)
	{
		FixPhyll = P * (1 - Rp * Math.Min(SowingDay - SDsa_sh, SDws));
	}
	else FixPhyll = P;
}
else
{
	if (SowingDay < SDsa_nh)
	{
		FixPhyll = P * (1 - Rp * Math.Min(SowingDay, SDws));
	}
	else FixPhyll = P;
}